types/collection.hh

```diff
@@ -139,3 +149,21 @@ collection_type_impl::pack_fragmented(BytesViewIterator start, BytesViewIterator
     }
     return out;
 }
+
+template <typename Iterator>
+requires requires (Iterator it) { {*it} -> std::convertible_to<managed_bytes_view>; }
+managed_bytes
+collection_type_impl::pack_fragmented(Iterator start, Iterator finish, int elements, cql_serialization_format sf) {
+    size_t len = collection_size_len(sf);
+    size_t psz = collection_value_len(sf);
+    for (auto j = start; j != finish; j++) {
+        len += managed_bytes_view(*j).size_bytes() + psz;
```

Comment:
If we had size_bytes(managed_bytes_view) and size_bytes(bytes_view), perhaps the duplication here won't be necessary. But if you plan to eliminate the duplication soon, we can tolerate it.

Reply:
Oh, 06f1f357 makes this function unused. I forgot to remove it.